sdk/dotnet/Inputs/ServiceLevelObjectiveTimeWindowRollingArgs.cs

```diff
@@ -12,15 +12,9 @@ namespace Pulumi.NewRelic.Inputs
 
     public sealed class ServiceLevelObjectiveTimeWindowRollingArgs : global::Pulumi.ResourceArgs
     {
-        /// <summary>
```

Comment:
Some more missed descriptions - why is this changing?

Reply:
While I can't speak to this specific doc, I'm assuming upstream doesn't have a label for each branch in their unrolled recursive structure, and our refined docs parsing is correctly identifying that upstream doesn't have a description here.